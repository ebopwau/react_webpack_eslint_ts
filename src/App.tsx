import React from 'react';

import { DelightfulComponent } from 'components';
import { GlobalStyle } from './Global.styled';

const App = () => (
  <>
    <GlobalStyle />
    <DelightfulComponent text="App!" />
  </>
);

export default App;
