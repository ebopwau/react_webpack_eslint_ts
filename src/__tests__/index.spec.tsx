import React from 'react';
import { mount, ReactWrapper } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from '../App';

describe('Test App Entry point', () => {
  it('should have an App! text', () => {
    const wrapper: ReactWrapper = mount(<App />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
