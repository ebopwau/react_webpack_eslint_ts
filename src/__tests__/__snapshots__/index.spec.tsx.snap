// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test App Entry point should have an App! text 1`] = `
.c0 {
  text-align: center;
}

<App>
  <Memo(l) />
  <DelightfulComponent
    text="App!"
  >
    <styled.div>
      <div
        className="c0"
      >
        App!
      </div>
    </styled.div>
  </DelightfulComponent>
</App>
`;
